packages/radix-ui-themes/src/components/code.css

```diff
@@ -123,10 +123,6 @@
         filter: var(--base-button-solid-high-contrast-hover-filter);
       }
     }
-    &:where(:focus-visible) .rt-Code {
-      outline: 2px solid var(--color-focus-root);
-      outline-offset: 2px;
-    }
```

Comment:
Stylelint didn't like the resulting selector (`:where(.rt-Link) .rt-Code:where(.rt-variant-solid):where(:focus-visible) .rt-Code` which also doesn't make sense).Wasn't really doing anything, probably a permutation of some original style that is not needed anymoroe. Did not hurt to remove it.